### ISSUE TITLE
The benchmark of VTM12.1

### ISSUE THREAD

**Issue by achel-x** [LabShuHangGU/DCAE#10]
Hi authors,
Thanks for your kind open-source. 
It referred that the results of VTM12.1 are achieved by CompressAI in your paper.
While I found there is no Technick results in CompressAI.
https://github.com/InterDigitalInc/CompressAI/tree/master/results

I wonder how did you calculate the results? Or could you share it?

ps, I noticed that the results in CompressAI about Kodac, it provide larger range of bitrate. 
    "bpp": [
      0.04824490017361111,
      **0.11249542236328125,
      0.24581654866536465,
      0.4905454847547744,
      0.8748101128472224,**
      1.43084971110026,
      2.3246070014105906,
      3.6326090494791665
    ],

Did you just use 4 points of them(**highlight**) to calculate the bdrate?

**Comment by achel-x** [comment]
Another question.
How many epochs you train the MS-SSIM ckpt ?

**Comment by galijipaifan** [comment]
Hi
1.For the results of VTM12.1, we used the data provided by the authors of LIC_TCM. The Kodak data comes from CompressAI, while the CLIC and Technick results are based on measurements taken by the LIC_TCM authors. You can refer to the following data:

CLIC:
0.112941829	31.27302275
0.153214253	32.28169014
0.203541168	33.30552546
0.269554344	34.37486457
0.351250729	35.43661971
0.450237162	36.46045503
0.571745009	37.50704225
0.699275006	38.37919826

Technick:
0.11677368 	31.37619822 
0.14390048 	32.09379821 
0.21067416 	33.64488003 
0.28475120 	34.80675613 
0.38491172 	36.04482595 
0.54141252 	37.49334891 

2.We use the 6 nearest points to calculate the bdrate. For the kodak, we use the bpp from 0.04824490017361111 to 1.43084971110026.
3.Due to time constraints, the model was not fully trained for MS-SSIM optimization. Therefore, the MS-SSIM results reported in the paper have room for improvement. For the number of training epochs, you can refer to the  [training details](https://github.com/LabShuHangGU/DCAE/issues/5) used for the MSE-oriented model.

**Comment by achel-x** [comment]
Thank you so much for in-time reply. The trainig process is burdensome for sure ><

I'm just curious about the training of MS-SSIM. I don't want to train it from scratch. 

I trained your model as you descrbe in the issue you mentioned. 100 epoch totally, 2 4090s, 80 epochs for 1e-4, 20 eps for 1e-5.
Can I finetune the last epoch of MSE ckpt maybe 5 epochs for MS-SSIM? dose it enough? I am not sure.
How did you do that?

**Comment by galijipaifan** [comment]
Directly fine-tuning the MSE-optimized model using an MS-SSIM loss will lead to performance degradation. I recommend training two new MS-SSIM-based models from scratch: one for high compression rates (lambda 8.73) and another for low compression rates (lambda 60.5). You can use a batch size of 8 for this, as its performance is only about 0.7% BD-rate worse than that of a batch size of 16.
